### DISCUSSION TITLE
Feature Request: Plan on supporting the OpenPodcastAPI

### DISCUSSION THREAD

**Discussion by vicerious001** [madeofpendletonwool/PinePods#313] [Ideas]
Currently in development by a coalition of podcast software developers including the teams behind AntennaPod, gpodder app for Nextcloud, Funkwhale, etc, the OpenPodcastAPI is/will be an open API for synchronizing podcast subscriptions, progress, queues, and so on.

https://openpodcastapi.org/

**Comment by madeofpendletonwool** [comment]
I absolutely have my eye on the progress of the API and will for sure be implementing it into Pinepods in the coming months. I'll keep this issue open as I make plans for API changes and post progress updates in here. Keep and eye out!

**Comment by madeofpendletonwool** [comment]
A significant amount of Podcasting 2.0 features are now supported including People, chapters, funding and transcript tags. I realize the OpenPodcastAPI is different but just giving an update on progress for supporting the new podcasting features.
